### ISSUE TITLE
Clarification needed about localstatedir

### ISSUE THREAD

**Issue by biochem-fan** [apptainer/apptainer-admindocs#165]
## Which Document page:

https://apptainer.org/docs/admin/main/installation.html#filesystem-support-limitations

### Expected results:

Requirements about `localstatedir` (and `SESSIONDIR`) are clearly stated.

### Actual results:

I am new to Apptainer and get confused about `localstatedir` (and `SESSIONDIR`). Clarification of the document would be very useful.

Suppose the `software` account (without `root` privileges) installs Apptainer to a shared file system of a cluster and wants to let other people use it.

1. What permission should the directory have? [The old Singularity documentation](https://docs.sylabs.io/guides/3.0/admin-guide/admin_quickstart.html#setting-localstatedir) says it must be 0755 and owned by `root:root` but the Apptainer document does not say anything.  
  [This GitHub issue comment](https://github.com/apptainer/apptainer/issues/1283#issuecomment-1507641190) says "It does not need to be writable". Does this still apply?
2. The same issue comment also says "Most important is that the directory has to pre-exist". If we specify `/var`, is it OK, because it is present on every node? Or do we have to manually make subdirectories (e.g. `/var/lib/apptainer/mnt/session`) on every node? (But this would require the `root` privileges; in such a case we could use `/tmp` instead).

**Comment by DrDaveD** [comment]
That github issue comment was referring to `/var/apptainer/mnt/session`, not to everything in `localstatedir`.  That directory needs to pre-exist and doesn't need to be writable.

All that documentation about `localstatedir` and installing on a network filesystem is quite old and I hadn't even noticed it before.  I'm not sure how accurate it is anymore or what they were even referring to.  I know that I have had great success with installing apptainer entirely in a read-only network filesystem, with nothing on the local node.  I even use `install-unprivileged.sh` to install relocatable pre-built binaries there.  The implications may be different if you need a setuid installation, but probably we should just delete that documentation to prevent it from misleading people.

**Comment by biochem-fan** [comment]
Thank you very much for your reply.

> I know that I have had great success with installing apptainer entirely in a read-only network filesystem, with nothing on the local node. I even use install-unprivileged.sh to install relocatable pre-built binaries there.

Yes, I also tried `install-unprivileged.sh` on a login node to install Apptainer on an NFS mounted user area without root privileges. It _seems_ the binary is usable by other user accounts and on other cluster nodes but I tested only the alpine image and am not sure if other functionalities also work. This is why I asked it here.

If developers suspect the paragraph is a false alarm, I will let our users try the binary and see if problems arise.
